cibuild.sh

```diff
@@ -174,11 +174,11 @@ function run_builds {
   local ncpus=`grep -c ^processor /proc/cpuinfo`
   local options="-j $ncpus"
 
-  if [ "X$build" = "Xcheck" ]; then
+  if [ "X$build" != "Xfull" ]; then
```

Comment:
Should we remove -b and support pass the mulitple file here?

Reply:
Ok, done